src/style.css

```diff
@@ -1 +1,56 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf), url(./fonts/Avenir-Heavy.ttf);
```

Comment:
It's more efficient to define multiple font weights in separate `@font-face` declarations, each with its own `font-weight` property, rather than combining them in a single `src` property.